database/postgres/index.js

```diff
@@ -0,0 +1,52 @@
+const { Pool, Client } = require('pg');
+// const { Review } = require('./schema_postgres.js')
+
+// mongoose.connect('mongodb://127.0.0.1:27017/test', { useNewUrlParser: true })
+//   .then(() => {
+//   })
+//   .catch(err => { // mongoose connection error will be handled here
+//       console.error('App starting error:', err.stack);
+//       process.exit(1);
+//    });
+
+const pool = new Pool({
+  user: 'root',
+  host: 'localhost',
+  database: 'reviews',
+  password: 'admin',
```

Comment:
In the future, you may want to add this into a config.js file to hide your password.